packages/ra-core/src/reducer/admin/resource/list/ids.js

```diff
@@ -61,25 +60,19 @@ export default resource => (previousState = [], { type, payload, meta }) => {
                 ...previousState.slice(index + 1),
             ];
 
-            Object.defineProperty(
-                newState,
-                'fetchedAt',
-                previousState.fetchedAt
-            );
+            Object.defineProperty(newState, 'fetchedAt', {
```

Comment:
Why using `defineProperty` instead of `newState.fetchedAt`?

Reply:
to make a non-enumerable propery